google/common_operation.go

```diff
@@ -109,6 +110,13 @@ func CommonRefreshFunc(w Waiter) resource.StateRefreshFunc {
 				return op, "done: false", nil
 			}
 
+			for _, e := range getAllTypes(err, &googleapi.Error{}, &url.Error{}) {
+				if isRetryableError(e, nil) {
```

Comment:
While you're here, do you mind making the 404 condition above a [retry predicate](https://github.com/terraform-providers/terraform-provider-google/blob/cf8fa58279d0b65132bc864448652c54429fc71a/google/error_retry_predicates.go) and passing it in to `isRetryableError`?

The reason we retry those 404's is that [operations don't always exist immediately](https://gist.github.com/rileykarson/cc3823cea8e333bd9f57dff34b0cc83c).